components/CollectionComponent copy.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useContext } from "react";
+import { GUITARS } from "../shared/guitars";
+import styles from "./Styles";
+import { Button, FlatList, View, Text } from "react-native";
+import GuitarModal from "./GuitarModal";
+import { GuitarContext } from "./GuitarContext";
+
+const Collection = ({ navigation }) => {
+    const [showAddGuitar, setShowAddGuitar] = useState(false);
+    const [gtrlist, setGtrlist] = useState(GUITARS);
+    const [selectedGuitar, setSelectedGuitar] = useState({});
```

Comment:
ok, trying to understand what you mean here. so is what you're saying that i need to account for if for some reason selectedGuitar is undefined? are you saying it is too simple right now because it assumes there will always be an object selected?

Reply:
Thinking this through again... So am I correct in assuming the initial state should be 'null', instead of {} ?

Reply:
Also in the current code (not the  'copy') I don't have a ### selectedGuitar constant, so I guess I'm not having to deal with that currently...